### PR TITLE
[rtlsdr] Add new port

### PR DESCRIPTION
**Pull request by ghost**
- What does your PR fix? Fixes issue #
Nothing, this adds a new port for **rtl-sdr** - software that turns a Realtek RTL2832 based DVB dongle into a SDR receiver (https://osmocom.org/projects/rtl-sdr)

- Which triplets are supported/not supported? Have you updated the CI baseline?
x86-windows and x64-windows are the only supported ones. Not yet.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes.

### CODE REVIEW

**Comment by JackBoosY** [comment]
Hi @echesakov, thanks for this PR!
Please sign CLA first.

Thanks.

**Comment by echesakov** [comment]
> Please sign CLA first.

@JackBoosY I did, can you please re-run the ci?

**Comment by JackBoosY** [comment]
```
fatal: unable to connect to git.osmocom.org:
git.osmocom.org[0: 78.46.96.155]: errno=Unknown error
```

File: ports/rtlsdr/CONTROL [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Source: rtlsdr
+Version: 2020-04-16
+Homepage: https://osmocom.org/projects/rtl-sdr
+Description: rtl-sdr is a library that turns your Realtek RTL2832 based DVB dongle into a SDR receiver
+Build-Depends: libusb, pthreads (windows)
+Supports: windows&(x86|x64)
```
Comment (JackBoosY) [inline ports/rtlsdr/CONTROL:6]: Please add the following codes to _VCPKG_PATH/scripts/ci.baseline.txt_:
```
rtlsdr:x64-uwp=fail
rtlsdr:arm64-windows=fail
rtlsdr:arm-uwp=fail
rtlsdr:x64-linux=fail
rtlsdr:x64-osx=fail
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
We firewall the git protocol at the moment (we allow only HTTP because we don't want a rogue PR to be able to turn our test fleet into spambots and that sort of thing). Is there an HTTP endpoint we can use or do we have to open that up?

**Comment by BillyONeal** [comment]
#10980 will open that port

**Comment by echesakov** [comment]
I have update the PR to use GitHub mirror https://github.com/osmocom/rtl-sdr

**Comment by strega-nil** [comment]
Alright, this looks good! Thanks @echesakov :)

**Pull request merged**